src/styles/index.scss

```diff
@@ -1,3 +1,105 @@
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+h3 {
+  margin: 0;
+}
+
+.cards {
+  box-sizing: border-box;
+  color: #060b35;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  border: 1px solid #f3f3f3;
+
+  &__foto {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+  &__code {
+    margin-top: 4px;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: #616070;
+  }
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+  &__price {
+    color: #616070;
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+    }
+    &-price {
+      color: #060b35;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+  &__button {
+    box-sizing: border-box;
+    color: #fff;
+    text-transform: uppercase;
+    background-color: #00acdc;
+    border-radius: 5px;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    margin-top: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    display: flex;
+    border: 1px solid #00acdc;
+
+    &:hover {
+      color: #00acdc;
+      background-color: #fff;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(/src/images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+  &--4 .stars__star:nth-child(-n + 4) {
```

Comment:
The selector `.stars--4 .stars__star:nth-child(-n + 4)` can be simplified. Since the `.stars__star` elements are already within a `.stars--4` block, you can use `&:nth-child(-n + 4)` inside the `.stars__star` block to apply the styles.